polynote-frontend/polynote/ui/component/kernel_ui.ts

```diff
@@ -53,21 +53,8 @@ export class KernelUI extends UIMessageTarget {
 
         this.socket.addEventListener('close', () => this.setKernelState('disconnected'));
 
-        this.socket.addMessageListener(messages.Error, (code, err) => {
-            console.log("Kernel error:", err);
-
-            const {el, messageStr, cellLine} = errorDisplay(err, this.path);
-
-            const id = "KernelError";
-            const message = div(["message"], [
-                para([], `${err.className}: ${err.message}`),
-                para([], el)
-            ]);
-            this.tasks.updateTask(id, id, message, TaskStatus.Error, 0);
-
-            // clean up (assuming that running another cell means users are done with this error)
-            this.socket.listenOnceFor(messages.CellResult, () => this.tasks.updateTask(id, id, message, TaskStatus.Complete, 100));
-        });
+        this.socket.addMessageListener(messages.Error, (code, err) => this.errorDisplay(code, err));
+        SocketSession.global.addMessageListener(messages.Error, (code, err) => this.errorDisplay(code, err));
```

Comment:
@jeremyrsmith looks like the socket-per-notebook change lost the global error display in the sidebar. I changed it back to this for now but we might want to consider a better separation between the notebook and server errors.